src/stan_math_backend/Stan_math_code_gen.ml

```diff
@@ -37,16 +37,19 @@ let stanc_args_to_print =
   |> List.filter ~f:sans_model_and_hpp_paths
   |> String.concat ~sep:" "
 
-let pp_unused = fmt "(void) %s;  // suppress unused var warning@ "
+let pp_unused = fmt "(void) %s;  // suppress unused var warning "
 
 (** Print name of model function.
   @param prog_name Name of the Stan program.
   @param fname Name of the function.
  *)
 let pp_function__ ppf (prog_name, fname) =
-  pf ppf "static const char* function__ = %S;@ "
-    (strf "%s_namespace::%s" prog_name fname) ;
-  pp_unused ppf "function__"
+  pf ppf {|
+    static const char* function__ = %S;
+    %s
+    |}
+    (strf "%s_namespace::%s" prog_name fname)
+    (strf "%a" pp_unused "function__")
```

Comment:
This is so that the `(void) function__;` is now properly indented.

Before this it was:
```cpp
    static const char* function__ = "simple_function_model_namespace::write_array";
(void) function__;  // suppress unused var warning
```